packages/cactus-cmd-api-server/package.json

```diff
@@ -43,14 +43,14 @@
     "dist/*"
   ],
   "scripts": {
-    "codegen": "run-p 'codegen:*'",
+    "codegen": "yarn run codegen:openapi && yarn run codegen:proto",
```

Comment:
@zondervancalvez Could you please explain why was this specific change necessary?

Reply:
After few tests, we revert this to original and added minor changes.